src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    private Random random = new Random();
+    private Color[] colors = Color.values();
```

Comment:
Consider using a more descriptive name for the `colors` field. For example `availableColors` would make the code easier to understand.

Reply:
Done it but I have a feeling that the shorter name, the easier it is to decode.